src/services/rollouts.js

```diff
@@ -122,4 +125,144 @@ module.exports = class RolloutsHelper {
 			throw error
 		}
 	}
+	/**
+	 * Rollout List
+	 * @method
+	 * @name list
+	 * @param {Object} req - request data.
+	 * @returns {JSON} - List of rollouts
+	 */
+	static async list(organization_id, loggedInUserId, queryParams, searchText = '', page, limit) {
+		try {
+			let result = {
+				data: [],
+				count: 0,
+			}
+			let filters = {
+				organization_id,
+				user_id: loggedInUserId,
+			}
+
+			if (searchText && searchText != '') {
+				filters.title = {
+					[Op.iLike]: '%' + searchText + '%',
+				}
+			}
+
+			if (queryParams.type && queryParams.type != '') {
+				filters.type = queryParams.type
+			}
+
+			if (queryParams.status && queryParams.status != '') {
```

Comment:
@adithyadinesh0412  type will be program or solution, solution u dont need to list . Remove solution type rollout and list